### DISCUSSION TITLE
Cannot See the Login Page

### DISCUSSION THREAD

**Discussion by pen1064** [kimai/kimai#3444] [Questions & Answers]
- Have been using the following setup, but when I tried to do "127.0.0.1:8010/kimai", the page cannot be loaded.
- I am wondering how can I fix it.....  

<img width="517" alt="image" src="https://user-images.githubusercontent.com/45325095/181694795-617ed798-47a4-4f47-8d2a-7f942bbd2189.png">

**Comment by kevinpapst** [comment]
location / is hardly the same as /kimai
